core/server/master/src/main/java/alluxio/master/block/BlockMasterClientServiceHandler.java

```diff
@@ -62,113 +62,95 @@ public BlockMasterClientServiceHandler(BlockMaster blockMaster) {
   @Override
   public void getBlockInfo(GetBlockInfoPRequest request,
       StreamObserver<GetBlockInfoPResponse> responseObserver) {
-
     long blockId = request.getBlockId();
     GetBlockInfoPOptions options = request.getOptions();
-
-    RpcUtils.call(LOG, (RpcUtils.RpcCallableThrowsIOException<GetBlockInfoPResponse>) () -> {
-
-      return GetBlockInfoPResponse.newBuilder()
-          .setBlockInfo(GrpcUtils.toProto(mBlockMaster.getBlockInfo(blockId))).build();
-    }, "CreateFile", "blockId=%s, options=%s", responseObserver, blockId, options);
+    RpcUtils.call(LOG,
+        () -> GetBlockInfoPResponse.newBuilder()
+            .setBlockInfo(GrpcUtils.toProto(mBlockMaster.getBlockInfo(blockId))).build(),
+        "GetBlockInfo", "blockId=%s, options=%s", responseObserver, blockId, options);
```

Comment:
getBlockInfo?

Reply:
I changed it all to capitalized to be consistent with our other logging.